features/google/src/views/ministories/placeVisits.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import analysisKeys from "../../model/analyses/analysisKeys.js";
+import { SingleDataStory } from "./singleDataStory.jsx";
+import FilterChipBubbles from "../../components/filterChipBubbles/filterChipBubbles.jsx";
+
+class PlaceVisitsMinistory extends SingleDataStory {
+    constructor(props) {
+        super(props);
+        this._neededAnalyses = [analysisKeys.groupedPlaceVisits];
+    }
+    get title() {
+        return "Place Visits";
```

Comment:
just ❓ ...don't we need translations in miniStories? I see they're missing at all the places

Reply:
Not in prototyping tech demos